Tests/SwerverTests/RouterSpec/RouterSpec.swift

```diff
@@ -9,13 +9,13 @@ class RouterSpec: QuickSpec {
             var router: Router!
             var formatter: ResponseFormatter!
             var responseBuilder: ResponseBuilder!
-            
+
             beforeEach {
                 router = Router()
                 formatter = ResponseFormatter()
                 responseBuilder = ResponseBuilder()
             }
-            
+
             context("when receive a GET request for an existing route") {
                 it("returns an response with 200 OK status") {
                     let aGetRequest = HttpRequest(method: RequestMethod.get, url: URL(string: "/demo")!, httpVersion: HttpVersion.current)
```

Comment:
Do you need to force unwrap the optional here?

Reply:
Xocde will throw errors if default value or force unwrapping isn't used in certain cases.